### PR TITLE
Functional tests - Add new test 'BO > Orders > Return order'

### PR DESCRIPTION
**Pull request by nesrineabdmouleh**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Add new test 'BO > Orders > Return order'
| Type?             | refacto
| Category?         | TE
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | Nightly & CI are :green_circle: 
 UI Tests          | https://github.com/nesrineabdmouleh/testing_pr/actions/runs/6497171670
| Fixed issue or discussion?     | no
| Related PRs       | no
| Sponsor company   |

### CODE REVIEW

File: tests/UI/pages/BO/orders/view/productsBlock.ts [hunk 1]
```diff
@@ -254,7 +265,7 @@ class ProductsBlock extends ViewOrderBasePage {
    * @param page {Frame|Page} Browser tab
    * @returns {Promise<number>}
    */
-  getProductsNumber(page: Frame|Page) : Promise<number> {
+  getProductsNumber(page: Frame | Page): Promise<number> {
```
Comment (Progi1984) [inline tests/UI/pages/BO/orders/view/productsBlock.ts]:
```suggestion
  async getProductsNumber(page: Frame | Page): Promise<number> {
```

**Review by Progi1984 — COMMENTED** [review]
A small feedback

File: tests/UI/pages/BO/orders/view/productsBlock.ts (hunk 1 above)
Comment (nesrineabdmouleh) [reply tests/UI/pages/BO/orders/view/productsBlock.ts]: Done :+1:

**Review by matthieu-rolland — APPROVED** [review]

**Review by Progi1984 — APPROVED** [review]

**Review by boubkerbribri — APPROVED** [review]

**Comment by boubkerbribri** [comment]
Thanks @nesrineabdmouleh

**Pull request merged**